src/Websites/Websites.Autorest/custom/New-AzStaticWebApp.ps1

```diff
@@ -0,0 +1,386 @@
+    # # To construct, see NOTES section for SKUCAPABILITY properties and create a hash table.
+    # ${SkuCapability},
+
+    # [Parameter()]
+    # [Microsoft.Azure.PowerShell.Cmdlets.Websites.Category('Body')]
+    # [System.Int32]
+    # # Default number of workers for this App Service plan SKU.
+    # ${SkuCapacityDefault},
+
+    # [Parameter()]
+    # [Microsoft.Azure.PowerShell.Cmdlets.Websites.Category('Body')]
+    # [System.Int32]
+    # # Maximum number of Elastic workers for this App Service plan SKU.
+    # ${SkuCapacityElasticMaximum},
+
+    # [Parameter()]
+    # [Microsoft.Azure.PowerShell.Cmdlets.Websites.Category('Body')]
+    # [System.Int32]
+    # # Maximum number of workers for this App Service plan SKU.
+    # ${SkuCapacityMaximum},
+
+    # [Parameter()]
+    # [Microsoft.Azure.PowerShell.Cmdlets.Websites.Category('Body')]
+    # [System.Int32]
+    # # Minimum number of workers for this App Service plan SKU.
+    # ${SkuCapacityMinimum},
+
+    # [Parameter()]
+    # [Microsoft.Azure.PowerShell.Cmdlets.Websites.Category('Body')]
+    # [System.String]
+    # # Available scale configurations for an App Service plan.
+    # ${SkuCapacityScaleType},
+
+    # [Parameter()]
+    # [Microsoft.Azure.PowerShell.Cmdlets.Websites.Category('Body')]
+    # [System.String]
+    # # Family code of the resource SKU.
+    # ${SkuFamily},
+
+    # [Parameter()]
+    # [Microsoft.Azure.PowerShell.Cmdlets.Websites.Category('Body')]
+    # [System.String[]]
+    # # Locations of the SKU.
+    # ${SkuLocation},
+
+    [Parameter()]
+    [Microsoft.Azure.PowerShell.Cmdlets.Websites.Category('Body')]
+    [System.String]
+    # Name of the resource SKU.
+    ${SkuName},
+
+    # [Parameter()]
+    # [Microsoft.Azure.PowerShell.Cmdlets.Websites.Category('Body')]
+    # [System.String]
+    # # Size specifier of the resource SKU.
+    # ${SkuSize},
+
+    # [Parameter()]
+    # [Microsoft.Azure.PowerShell.Cmdlets.Websites.Category('Body')]
+    # [System.String]
+    # # Service tier of the resource SKU.
+    # ${SkuTier},
+
+    [Parameter()]
+    [ArgumentCompleter([Microsoft.Azure.PowerShell.Cmdlets.Websites.Support.StagingEnvironmentPolicy])]
+    [Microsoft.Azure.PowerShell.Cmdlets.Websites.Category('Body')]
+    [Microsoft.Azure.PowerShell.Cmdlets.Websites.Support.StagingEnvironmentPolicy]
+    # State indicating whether staging environments are allowed or not allowed for a static web app.
+    ${StagingEnvironmentPolicy},
+
+    [Parameter()]
+    [Microsoft.Azure.PowerShell.Cmdlets.Websites.Category('Body')]
+    [Microsoft.Azure.PowerShell.Cmdlets.Websites.Runtime.Info(PossibleTypes=([Microsoft.Azure.PowerShell.Cmdlets.Websites.Models.Api20201201.IResourceTags]))]
+    [System.Collections.Hashtable]
+    # Resource tags.
+    ${Tag},
+
+    [Parameter()]
+    [Microsoft.Azure.PowerShell.Cmdlets.Websites.Category('Body')]
+    [System.String]
+    # Description of the newly generated repository.
+    ${TemplatePropertyDescription},
```

Comment:
We should rename these parameters about template:
| old                                     | new                         |
| --------------------------------------- | --------------------------- |
| `TemplatePropertyDescription`           | `ForkRepositoryDescription` |
| `TemplatePropertyIsPrivate`             | `ForkRepositoryIsPrivate`   |
| `TemplatePropertyOwner`                 | `ForkRepositoryOwner`       |
| `TemplatePropertyRepositoryName`        | `ForkRepositoryName`        |
| `TemplatePropertyTemplateRepositoryUrl` | `TemplateRepositoryUrl`     |

Reply:
Done